build/references.py

```diff
@@ -171,8 +172,33 @@ def get_standard_citatation(citation, cache, override):
 template = jinja2.Template(converted_text)
 converted_text = template.render(**stats)
 
+all_sections_file = gen_dir.joinpath('all-sections.md')
+
+#FIXME: Stubbing metadata:
+title_text = "Manubot Rootstock: Manuscript Title"
```

Comment:
No metadata should be hardcoded in `references.py`. Modify `references.py` sparingly. Instead could you create a `content/metadata.yml` file, with the hardcoded data. Then author information could be inserted into the yaml structure before it's put into `all_sections.md`?